fast64_internal/f3d/f3d_writer.py

```diff
@@ -1247,7 +1247,10 @@ def saveOrGetF3DMaterial(material, fModel, obj, drawLayer, convertTextureData):
 		defaultRM = None
 
 	defaults = bpy.context.scene.world.rdp_defaults
-	saveGeoModeDefinition(fMaterial, f3dMat.rdp_settings, defaults, fModel.matWriteMethod)
+	if bpy.context.scene.f3d_type == 'F3DEX_GBI_2':
+		saveGeoModeDefinitionGBI2(fMaterial, f3dMat.rdp_settings, defaults, fModel.matWriteMethod)
```

Comment:
I would call it f3dex2 instead of gbi2 but I'm not very familiar with microcodes so not so sure